frontend/src/features/admin-form/settings/components/PaymentSettingsSection/EnableGSTSection.tsx

```diff
@@ -0,0 +1,49 @@
+import { useCallback, useMemo } from 'react'
+import { Skeleton } from '@chakra-ui/react'
+
+import { FormResponseMode } from '~shared/types'
+
+import Toggle from '~components/Toggle'
+
+import { useMutateFormSettings } from '../../mutations'
+import { useAdminFormSettings } from '../../queries'
+
+export const FormGSTToggle = (): JSX.Element => {
```

Comment:
Component name should follow the filename (or vice versa)

Reply:
fixed too [312e39d](https://github.com/opengovsg/FormSG/pull/6538/commits/312e39d81697e1c2bf4e9bf74ded01c1fa7ac09b)